packages/site-parsers/src/parsers/wix/components/sound-cloud.js

```diff
@@ -10,9 +10,12 @@ module.exports = {
 			themeData.style.properties &&
 			themeData.style.properties.param_font_resolveUrl
 		) {
-			return createBlock( 'core/embed', {
-				url: themeData.style.properties.param_font_resolveUrl,
-			} );
+			const url = themeData.style.properties.param_font_resolveUrl.replace(
+				/^(")|(")$/g,
```

Comment:
Just curious, why the parentheses?

Reply:
Hmm, I didn't see that I left the parentheses. Initially, I wanted to match and replace by group and then figure out that I don't need to complicate (I forgot to remove it). :)

Reply:
Extra parentheses have been removed.